### PR TITLE
RockPi4 TRS build instructions

### PR DESCRIPTION
**Pull request by petegriffin**
Now we have a TRS build that integrates ewaol with ledge security features. Update the build instructions to remove references to LEDGE RP and include the new TRS build and flashing instructions.

### CODE REVIEW

**Comment by petegriffin** [comment]
@jbech-linaro fyi some updated build instructions for rockpi4b

**Comment by jbech-linaro** [comment]
> @jbech-linaro fyi some updated build instructions for rockpi4b

Thanks @petegriffin , I'll review on Monday.

File: docs/install/qemu.rst [hunk 1]
```diff
@@ -3,9 +3,10 @@
 QEMU
 ####
 
-LEDGE Reference Platform (ledge-os-manifest) is a Linux distro based on
-Yocto/OpenEmbedded. It also uses firmware from Linaro :ref:`Trusted Substrate`
-(meta-ts). This document describes how to build and run for target QEMU.
+TRS is a Linux distro based on Yocto/OpenEmbedded, it builds on top of
+Arms EWAOL and adds extra security features. It also uses firmware from
+Linaro :ref:`Trusted Substrate` (meta-ts). This document describes how
+to build and run for QEMU tarrget.
```
Comment (mathieupoirier) [inline docs/install/qemu.rst:9]: I think this should be something like: 

"TRS is a Linux distribution based on Yocto/OpenEmbedded that builds on top of firmware from Linaro and incorporates elements such as Arm's EWAOL and additional security features."  

The same applies to the other installation description.  I will add more comments as I follow the instructions.

Comment (jbech-linaro) [reply docs/install/qemu.rst:9]: I'd suggest to move this into the "About" section instead. I already have #4 that is related to this.

Comment (jbech-linaro) [reply docs/install/qemu.rst:9]: s/tarrget/target/.

File: docs/install/rockpi4.rst [hunk 2]
```diff
@@ -107,8 +100,8 @@ Restart
 Power cycle board it and it has to boot automatically now.
 
 .. note::
-	First boot with a fresh root fs is slow, please wait for a couple of minutes
-	(really) when no output is shown:
+	First boot with a fresh root fs is quite slow, please wait for a couple
```
Comment (jbech-linaro) [inline docs/install/rockpi4.rst:103]: s/first/second/ I believe. First boot wasn't too bad, it's the second boot after running the efi-commands that seem to be slow.

**Comment by jbech-linaro** [comment]
Minor comments, with those addressed:
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: docs/install/qemu.rst (hunk 1 above)
Comment (petegriffin) [reply docs/install/qemu.rst:9]: fixed

File: docs/install/rockpi4.rst (hunk 2 above)
Comment (petegriffin) [reply docs/install/rockpi4.rst:103]: fixed

Comment (petegriffin) [reply docs/install/rockpi4.rst]: yes you're correct it is the second boot when it encrypts the rootfs. I've updated in the latest iteration and added your reviewed-by

**Review by jbech-linaro — APPROVED** [review]

**Pull request merged**